inc/sitemaps/class-sitemap-image-parser.php

```diff
@@ -425,6 +393,92 @@ protected function get_absolute_url( $src ) {
 	}
 
 	/**
+	 * Returns the attachments for a gallery.
+	 *
+	 * @param int   $id      The post id.
+	 * @param array $gallery The gallery config.
+	 *
+	 * @return array The selected attachments
+	 */
+	protected function get_gallery_attachments( $id, $gallery ) {
+
+		// When there are posts to include.
```

Comment:
posts in this context seems to refer to posts with the post_type attachments, it might be clearer to just call them attachments.